.circleci/config.yml

```diff
@@ -67,100 +58,81 @@ commands:
 
 jobs:
   build:
+    docker:
+      - image: planetariumhq/9c-launcher-circleci:git-6fa331fdef32d6f124d06b6c48d07e352cc11067
+        auth:
+          username: $DOCKER_USERNAME
+          password: $DOCKER_PASSWORD
+    resource_class: large
     parameters:
       os:
-        type: executor
-    executor: << parameters.os >>
+        type: string
     steps:
       - when:
           condition:
-            equal: [*windows, << parameters.os >>]
+            equal: [ windows, << parameters.os >> ]
           steps:
             - run:
                 command: |
-                  git config --global core.symlinks true # https://github.com/git-for-windows/git/wiki/Symbolic-Links 
-                  nvm install lts
-                  rm "/c/Program Files/nodejs" -rf
-                  nvm use 16.16.0
-      - checkout-update-submodules:
-          paths: "NineChronicles.Headless"
-      - yarn-install
+                  echo "export OS_ALIAS=win32" >> $BASH_ENV
+                  echo "export ARTIFACT=Windows.zip" >> $BASH_ENV
+                  echo "export EXECUTABLE=." >> $BASH_ENV
+                  echo "export ZIP_CMD='7zr a -r'" >> $BASH_ENV
+                name: Setup Windows OS Variables
       - when:
           condition:
-            or:
-              - equal: [*macos, << parameters.os >>]
-              - equal: [*linux, << parameters.os >>]
+            equal: [ linux, << parameters.os >> ]
           steps:
             - run:
                 command: |
-                  curl https://dotnet.microsoft.com/download/dotnet/scripts/v1/dotnet-install.sh -o "dotnet-install.sh"
-                  chmod +x dotnet-install.sh
-                  ./dotnet-install.sh --channel Current
-                  echo 'export PATH=$PATH:$HOME/.dotnet' >> $BASH_ENV
-                name: Install .NET Core
+                  echo "export OS_ALIAS=linux" >> $BASH_ENV
+                  echo "export ARTIFACT=Linux.tar.gz" >> $BASH_ENV
+                  echo "export EXECUTABLE=resources/app/9c" >> $BASH_ENV
+                  echo "export ZIP_CMD='tar cvfz'" >> $BASH_ENV
+                name: Setup Linux OS Variables
       - when:
           condition:
-            equal: [*macos, << parameters.os >>]
+            equal: [ macos, << parameters.os >> ]
           steps:
             - run:
                 command: |
-                  curl "https://awscli.amazonaws.com/AWSCLIV2.pkg" -o "AWSCLIV2.pkg"
-                  sudo installer -pkg AWSCLIV2.pkg -target /
-                name: Install AWS CLI
+                  echo "export OS_ALIAS=darwin" >> $BASH_ENV
+                  echo "export ARTIFACT=MacOS.tar.gz" >> $BASH_ENV
+                  echo "export EXECUTABLE='Nine Chronicles.app/Contents/Resources/app/9c.app/Contents/MacOS/9c'" >> $BASH_ENV
+                  echo "export ZIP_CMD='tar cvfz'" >> $BASH_ENV
+                name: Setup Mac OS Variables
```

Comment:
That's awesome trick! You might want to split this off to another script like `configure_artifact.sh` with conditionals.

```bash
if [ $0 = "macos" ]; then
	echo "export OS_ALIAS=darwin" >> $BASH_ENV
  echo "export ARTIFACT=MacOS.tar.gz" >> $BASH_ENV
  echo "export EXECUTABLE='Nine Chronicles.app/Contents/Resources/app/9c.app/Contents/MacOS/9c'" >> $BASH_ENV
  echo "export ZIP_CMD='tar cvfz'" >> $BASH_ENV
else
  echo "export OS_ALIAS=linux" >> $BASH_ENV
  echo "export ARTIFACT=Linux.tar.gz" >> $BASH_ENV
  echo "export EXECUTABLE=resources/app/9c" >> $BASH_ENV
  echo "export ZIP_CMD='tar cvfz'" >> $BASH_ENV
fi
```